### PR TITLE
Arm64 by default

### PR DESCRIPTION
**Pull request by ensary**
Request arm64 based build agent, run all CI on amd64 and arm64, use compatible shellcheck version.

### CODE REVIEW

**Review by mikerott — DISMISSED** [review]
😍

**Review by akhalina — DISMISSED** [review]

File: Makefile [hunk 1]
```diff
@@ -13,12 +13,13 @@ ARTIFACT := $(REGISTRY)$(IMAGE_PATH)/$(IMAGE_NAME)
...
 
 lint:
-	docker run --rm -i -v "$(DIR):/mnt:ro" koalaman/shellcheck:v0.6.0 commands/*
+	docker run --rm -i -v "$(DIR):/mnt:ro" koalaman/shellcheck:v0.8.0 commands/*
 
 test:
 	docker build -t local/test/sdcli .
 	docker build -t local/test/sdclitests test
-	docker run -it local/test/sdclitests
+	docker run -i local/test/sdclitests
```
Comment (dkolbly) [inline Makefile:21]: I'm surprised to see `-i` here; is it needed?

File: Makefile [hunk 2]
```diff
@@ -30,4 +31,4 @@ build:
 	docker build -t $(ARTIFACT) .
 
 run:
-	docker run -ti $(ARTIFACT)
+	docker run -i $(ARTIFACT)
```
Comment (dkolbly) [inline Makefile:34]: ditto `-i` question, though it's easier to imagine its utility here.

**Review by dkolbly — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (ensary) [reply Makefile:21]: The pytest formatter seems to be happier with `-i`

File: Makefile (hunk 2 above)
Comment (ensary) [reply Makefile:34]: Good question, I remove `t` but I will check why `i` is still there

File: docker-apt.list [hunk 3]
```diff
@@ -1 +1 @@
-deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/debian bullseye stable
+deb [signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/debian bullseye stable
```
Comment (gcase555) [inline docker-apt.list:1]: nice, didnt realize we didnt have to specify the arch here. makes multi-arch much easier to build.

**Review by gcase555 — APPROVED** [review]

**Review by akhalina — DISMISSED** [review]

**Review by akhalina — APPROVED** [review]

**Review by gcase555 — APPROVED** [review]

**Pull request merged**